logviewer/src/main/res/menu/menu_categories.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item android:id="@+id/actionSearch"
+        android:title="@string/action_search"
+        android:icon="@android:drawable/ic_menu_search"
+        app:showAsAction="always"
+        app:actionViewClass="androidx.appcompat.widget.SearchView" />
+
+    <item
+        app:showAsAction="ifRoom"
+        android:icon="@drawable/ic_share"
+        android:title="@string/action_share">
+
+        <menu>
+            <item
+                android:id="@+id/actionShareDb"
```

Comment:
Related to the other comments, at this point we shouldn't care if what we're sharing is a database, an XML file or something else, as long as it's some kind of a log dump file. 

Hence we could name this something like `actionShareLogDump`. Please also adjust the title string in the same way (both the resource name and the actual string).

Reply:
Fixed